app/src/main/java/com/example/posebook/mapReviewFragment.kt

```diff
@@ -0,0 +1,51 @@
+package com.example.posebook
+
+import android.os.Bundle
+import android.view.Gravity
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.RatingBar
+import android.widget.TextView
+import android.widget.Toast
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+
+
+class mapReviewFragment(var locationTitle: String, var locationSubTitle: String) :
```

Comment:
Just some minor nitpicking, a class name should start with capital letter (MapReviewFragment)